Composer/packages/client/src/recoilModel/dispatchers/__tests__/project.test.tsx

```diff
@@ -480,35 +480,36 @@ describe('Project dispatcher', () => {
     expect(renderedComponent.current.botStates.oneNoteSync).toBeUndefined();
   });
 
-  it('should be able to add a new skill to Botproject', async () => {
-    const skillId = projectId;
-    await act(async () => {
-      (httpClient.put as jest.Mock).mockResolvedValueOnce({
-        data: mockProjectResponse,
-      });
-      await dispatcher.openProject('../test/empty-bot', 'default');
-    });
-
-    const newProjectDataClone = cloneDeep(mockProjectResponse);
-    newProjectDataClone.botName = 'new-bot';
-    await act(async () => {
-      (httpClient.post as jest.Mock).mockResolvedValueOnce({
-        data: newProjectDataClone,
-      });
-      await dispatcher.addNewSkillToBotProject({
-        name: 'new-bot',
-        description: '',
-        schemaUrl: '',
-        location: '/Users/tester/Desktop/samples',
-        templateId: 'InterruptionSample',
-        locale: 'us-en',
-      });
-    });
-
-    expect(renderedComponent.current.botStates.newBot).toBeDefined();
-    expect(renderedComponent.current.botStates.newBot.botDisplayName).toBe('new-bot');
-    expect(navigateTo).toHaveBeenLastCalledWith(`/bot/${projectId}/skill/${skillId}/dialogs/emptybot-1`);
-  });
+  // TODO update to new creation flow
```

Comment:
TODO

Reply:
I have removed this test as it covers unused code. We have a task going tracking unit tests we need to create for new creation here 
https://github.com/microsoft/BotFramework-Composer/issues/7885